models/Room.js

```diff
@@ -45,17 +45,23 @@ class Room {
      */
     removePlayer(playerId) {
         var updatedRoomMembers, counter
-        this.udpConnections.delete(playerId)
-        this.roomMembers.forEach((roomMember, index, object) => {
-            if (roomMember.playerId === playerId) {
-                object.splice(index, 1)
-                updatedRoomMembers = object
+        try {
+            this.udpConnections.delete(playerId)
+            this.roomMembers.forEach((roomMember, index, object) => {
+                if (roomMember.playerId === playerId) {
+                    object.splice(index, 1)
+                    updatedRoomMembers = object
+                }
+            })
+            for (counter = 0; counter < updatedRoomMembers.length; counter++) {
+                updatedRoomMembers[counter].playerIndex = counter
             }
```

Comment:
I still don't quite understand the point of playerIndex. Is this needed? If it is used for getting the "first" user for ownership, we can just grab any member from the updateRoomMembers

Reply:
As we discussed before, we hardcode playerIndex = 0 in multiple sample apps. Will remove it to a random number within the roomMembers.length range in the next release.
Thanks for the suggestion.